src/transformers/models/beit/feature_extraction_beit.py

```diff
@@ -58,6 +64,9 @@ class BeitFeatureExtractor(FeatureExtractionMixin, ImageFeatureExtractionMixin):
             The sequence of means for each channel, to be used when normalizing images.
         image_std (:obj:`List[int]`, defaults to :obj:`[0.5, 0.5, 0.5]`):
             The sequence of standard deviations for each channel, to be used when normalizing images.
+        reduce_zero_label (:obj:`bool`, `optional`, defaults to :obj:`False`):
```

Comment:
Not super fan of the name as it does not reduce the zero label but all labels. `decrease_labels` or `decrement_labels` feels like a better name.